lib/sequent/core/event_store.rb

```diff
@@ -62,7 +62,7 @@ def load_events_for_aggregates(aggregate_ids)
         streams = Sequent.configuration.stream_record_class.where(aggregate_id: aggregate_ids)
 
         query = aggregate_ids.uniq.map { |aggregate_id| aggregate_query(aggregate_id) }.join(" UNION ALL ")
-        events = Sequent.configuration.event_record_class.connection.select_all(query).map! do |event_hash|
+        events = Sequent.configuration.event_record_class.connection.select_all(query).map do |event_hash|
```

Comment:
@lvonk Do you know why this was a `.map!`, which is deprecated now. The only reason I can think of is a slight memory optimization, but am I missing anything?

Reply:
Yes I think so too